src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,25 @@
+//#region Color variables
+$main-color: #060b35;
+$secondary-color: #616070;
+$active-color: #00acdc;
+$background-color: #fff;
+// #endregion
+
+$header-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+// #region Catalog sizes
+$card-margin-right: 48px;
+$card-margin-bottom: 46px;
+
+$card-size: 200px;
+$cards-count: 4;
+$star-size: 16px;
+
+$catalog-size:
```

Comment:
I think it's better to create a function and move it to another file

Reply:
i thought about it. That's correct suggestion, but if i do so, lint test won't pass because of too long function line.
I've decided to not create separate file for just one lonely function :)